Update README.md

Luci-d opened the pull request:
changed or removed 404's and added cool service IMHO

Luci-d committed:
Update README.md

changed or removed 404's and added cool service IMHO

dloss commented:
Thank you. Please keep the existing libraries and just update the links:

- https://hachoir.readthedocs.io/en/latest/
- https://github.com/dirtbags/py-pcap

Luci-d commented:
> Thank you. Please keep the existing libraries and just update the links:
> 
> * https://hachoir.readthedocs.io/en/latest/
> * https://github.com/dirtbags/py-pcap

Got it

Luci-d committed:
UPD

updated links
https://hachoir.readthedocs.io/en/latest/
https://github.com/dirtbags/py-pcap

Luci-d commented:
> Thank you. Please keep the existing libraries and just update the links:
> 
> * https://hachoir.readthedocs.io/en/latest/
> * https://github.com/dirtbags/py-pcap

Done

dloss commented:
Thank you for following up on this
